draft-packages/table/KaizenDraft/Table/Table.tsx

```diff
@@ -377,7 +387,8 @@ export const TableRow: React.VFC<TableRowProps> = ({
  *        shrink, and basis, due to IE11 compatibility. eg. use "1 1 auto"
  *        instead of just "1".
  */
-type TableRowCellProps = {
+interface TableRowCellProps
+  extends OverrideClassName<HTMLAttributes<HTMLSpanElement>> {
```

Comment:
```suggestion
  extends OverrideClassName<HTMLAttributes<HTMLElement>> {
```

The props don't appear to be spread into a `span`, but either an `a` or `div`. `HTMLElement` is the most generic :)

Reply:
Hey @HeartSquared, thanks for your feedback. I've made those changes if you'd like to re-review.